src/main/java/picard/sam/SamErrorMetric/BaseCalculator.java

```diff
@@ -28,8 +28,8 @@
 import htsjdk.samtools.util.SamLocusIterator;
 
 /**
- * An interface that can take a collection of bases (provided as {@link htsjdk.samtools.util.SamLocusIterator.RecordAndOffset RecordAndOffset}
- * and {@link SamLocusAndReferenceIterator.SAMLocusAndReference SAMLocusAndReference}) and generates a
+ * An interface that can take a collection of bases (provided as {@link htsjdk.samtools.util.SamLocusIterator.RecordAndOffset}
+ * and {@link htsjdk.samtools.reference.SamLocusAndReferenceIterator.SAMLocusAndReference}) and generates a
```

Comment:
The fully qualified name shouldn't be required here since the file imports `htsjdk.samtools.reference.SamLocusAndReferenceIterator`